tests/Unleash.Tests/Integration/TestCaseVariant.cs

```diff
@@ -4,15 +4,9 @@ namespace Unleash.Tests.Specifications
 {
     public class TestCaseVariant
     {
-        private Variant _expectedResult;
-
         public string Description { get; set; }
         public UnleashContextDefinition Context { get; set; }
         public string ToggleName { get; set; }
-        public Variant ExpectedResult
```

Comment:
This wasn't loading feature enabled correctly. So the spec tests haven't been checking feature_enabled correctly until now